pygromos/files/coord/cnf.py

```diff
@@ -848,13 +848,16 @@ def write_refpos(self, out_path: str) -> str:
     """
         convert file: 
     """
-    def createRDKITconf(self, mol:Chem.rdchem.Mol):
+    def createRDKITconf(self, mol:Chem.rdchem.Mol, conversionFactor:float = 0.1):
         """creates a PyGromosTools CNF type from a rdkit molecule. If a conformation exists the first one will be used.
 
         Parameters
         ----------
         mol : Chem.rdchem.Mol
             Molecule, possibly with a conformation
+
+        conversionFactor  :  float
+            the factor used to convert length from rdkit to Gromos
```

Comment:
maybe we should mention its angstroem -> nano meters

Reply:
agreed